apps/web-portal-client/lib/components/nav-bar/NavBar.jsx

```diff
@@ -26,10 +25,7 @@ export function NavBar() {
             />
           </div>
           <div className="flex items-center space-x-1">
-            <LoginButton
-              label={isLogged ? 'Logout' : 'Login'}
-              onClick={() => setIsLogged(!isLogged)}
-            />
+            <LoginButton />
             <MobileMenuButton
               shape={showMobileMenu ? 'M6 18L18 6M6 6l12 12' : 'M4 6h16M4 12h16M4 18h16'}
```

Comment:
give a proper names over variables to shapes

Reply:
done